server/app.js

```diff
@@ -272,6 +272,53 @@ app.get('/print/:id', asyncHandler(async (req, res, next)=>{
 	return next();
 }));
 
+//Account Page
+app.get('/ui/account', asyncHandler(async (req, res, next)=>{
+	const brew = {};

```

Comment:
We can probably rename this variable `brew` here to not confuse it with the way we use `brew` everywhere else, since in this case it is really just acting as a container for the specific data we plan to show on this page. It would add another prop passed along in the "render the page" section below, but probably makes more sense to call it `pageData` or `contents` or something dumb. Then we can have `contents.uiItems` or whatever.

Reply:
Renamed variable in this section to `data`